config/registry_image_pruner/cronjob.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: image-pruner-cronjob
+spec:
+  schedule: "0 0 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: image-pruner
+            image: registry.redhat.io/rhel8/python-39
```

Comment:
bind to tag and sha

Reply:
binded to the newest version tag available